source/platform/renderers/DirectX11/DirectX11.cpp

```diff
@@ -199,75 +183,90 @@ Font::Status DirectX11ResourceLoader::LoadFont(Font& font)
         return font.status = Font::Status::ErrorFileNotFound;
     }
 
-    SelectObject(hDC, hBitmap);
     SelectObject(hDC, hFont);
-
-    SetTextColor(hDC, RGB(0xFF, 0xFF, 0xFF));
-    SetBkColor(hDC, 0x00000000);
     SetTextAlign(hDC, TA_TOP);
 
-    float x = 0, y = 0;
     wchar_t str[2] = L"x";
-    SIZE sz;
+    std::vector<XMFLOAT4> sizes;
+    float spacing;
+    {
+        float x = 0, y = 0;
+        SIZE sz;
 
-    GetTextExtentPoint32W(hDC, L" ", 1, &sz);
+        GetTextExtentPoint32W(hDC, L" ", 1, &sz);
 
-    float spacing = sz.cx;
+        spacing = sz.cx;
 
-    std::vector<XMFLOAT4> sizes;
+        for (wchar_t c = BeginCharacter; c <= LastCharacter; c++)
+        {
+            str[0] = c;
+            GetTextExtentPoint32W(hDC, str, 1, &sz);
 
-    for (wchar_t c = L' '; c < L'~' + 1; c++)
-    {
-        str[0] = c;
-        GetTextExtentPoint32W(hDC, str, 1, &sz);
+            if (x + sz.cx > texWidth)
+            {
+                x = 0;
+                y += sz.cy;
+            }
 
-        if (x + sz.cx > texWidth)
-        {
-            x = 0;
-            y += sz.cy;
-        }
+            x = ceilf(x);
+            y = ceilf(y);
 
-        x = ceilf(x);
-        y = ceilf(y);
+            sizes.push_back(XMFLOAT4(x, y, x + sz.cx, y + sz.cy));
```

Comment:
If you `sizes.reserve(N)` outside the loop it will avoid the vector resizing on push. Not a biggie, but a good habit.